modules/installation/pages/proxy-setup.adoc

```diff
@@ -194,46 +194,72 @@ filename: "pxelinux.0"
 [[replace-susemgrproxy]]
 == Replace a {productname} Proxy
 
-A proxy does not contain any information about the clients that are connected to it.
-Therefore, a proxy can be replaced by a new one at any time.
+You can replace a proxy at any time, as it does not store any information about the clients that are connected to it.
+This process is handled using a reactivation key, which prevents you from losing the history of the proxy.
+If you do not use a reactivation key, the replacement proxy will become a new one with a new ID.
 The replacement proxy must have the same name and IP address as its predecessor.
 
+You can also re-register a proxy to change it from a traditional proxy to a Salt proxy.
+
+
+[IMPORTANT]
+====
+During the installation of the proxy, clients will not be able to reach the {productname} Server.
+After you have deleted a proxy, the systems list can be temporarily incorrect.
+All clients that were previously connected to the proxy will show as being directly connected to the server instead.
+After the first successful operation on a client, such as execution of a remote command or installation of a package or patch, this information will automatically be corrected.
+This may take some hours.
+====
```

Comment:
?? Where is this change coming from? Are you sure about this?

Reply:
It was already extant in the text, I just moved it. Can we verify it somehow?

Reply:
If it was there, I have no issue with it. I just didn't see it and thought it was new text.